reference/libsql-urls.md

```diff
@@ -41,22 +46,56 @@ instances over time depending on how Fly.io observes network latency.
 This URL always connects to a specific [instance] of a Turso database. You may
 want to use this in order to bypass the automatic routing provided by the
 logical database URL. Instance URLs appear for each instance in the output of
-`turso db show $DBNAME --instance-urls`.
+`turso db show $DBNAME --instance-urls`. Database instance URLs have the
+following format:
+
+```
+libsql://[RANDOM-CHARS]-[DB-NAME]-[ORG-NAME].turso.io
+```
+
+## Network protocol
+
+libSQL client libraries are free to choose what they consider to be the best (or
```

Comment:
client libraries are free to choose what they consider the best -- _when using libsql URLs_, no ?

Reply:
Clarified.